vcd/config_test.go

```diff
@@ -832,3 +880,27 @@ func importStateIdVmObject(orgName, vdcName, vappName, vmName, objectIdentifier
 			objectIdentifier, nil
 	}
 }
+
+func setBoolFlag(variable *bool, name, envVar, help string) {
+	if envVar != "" && os.Getenv(envVar) != "" {
+		*variable = true
+	}
```

Comment:
In govcd you have a `return` in this place. Here - no return.

Reply:
The return was wrong in both places. I found out after you suggested checking what happens when we use both env variable and command line flag.
It was removed from govcd as well